stable_baselines3/common/vec_env/vec_normalize.py

```diff
@@ -222,7 +222,11 @@ def reset(self) -> Union[np.ndarray, Dict[str, np.ndarray]]:
         self.old_obs = obs
         self.ret = np.zeros(self.num_envs)
```

Comment:
Nitpick, but this line could be removed. Also docs should note that reward statistics are now fixed (does not append extra zeros, as it previously did with this line)

Reply:
why can it be removed? because it is reset when `done=True`?
This work unless we call `.reset()` earlier.
I was also thinking about renaming it to `self.returns` and deprecating `self.ret`, what do you think?

Reply:
Oh gosh dangit, I somehow missed the whole "self" part of the variable. ^^.

And yup! Agree that it should be renamed to `self.returns`.